core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1603,7 +1602,8 @@ public Label getLabel(String expr) {
      * Returns the label atom of the given name.
      * @return non-null iff name is non-null
      */
-    public @Nullable LabelAtom getLabelAtom(@CheckForNull String name) {
+    @CheckForNull
+    public LabelAtom getLabelAtom(@CheckForNull String name) {
```

Comment:
Name can be passed not from obvious variables, i.e. from method that may return null in some cases. This should be automatically checked by analysing tools and not by human tracing.